devp2p/src/cli.rs

```diff
@@ -0,0 +1,41 @@
+use std::env;
+use std::ffi::OsString;
+use structopt::StructOpt;
+
+#[derive(StructOpt, Debug, PartialEq)]
+#[structopt(
+    name = "trin-devp2p2",
+    version = "0.0.1",
+    about = "Testing framework for portal network peer-to-peer network calls"
```

Comment:
I'm happy with `trin-peertest`, I'm assuming we are following the same crate/folder naming convention and set the folder name same as the crate name? It is funny because it looks strange to me to have dozen of folders starting with `trin-` in the repo but it is probably just me :smile:

Reply:
`trin-peertest` sounds good to me as well. Minor thing, but I second @ogenev on this one, having `trin` at the start of every crate seems repetitive. Also, if we're designing the framework with interoperability in mind, I think it would be best to take out `trin` to avoid confusion.

Reply:
> `trin-peertest` sounds good to me as well. Minor thing, but I second @ogenev on this one, having `trin` at the start of every crate seems repetitive. Also, if we're designing the framework with interoperability in mind, I think it would be best to take out `trin` to avoid confusion.

Right, I'm new enough to rust that I don't know the conventions here. My thought is that maybe it's okay to call the folder just `state` but to name the crate `trin-state` (assuming that's the name you see in crates.io). With `peertest`, and the idea being to test multiple clients eventually, maybe it should be named `portal-peertest` or `ethportal-peertest`, or something like that.

Reply:
> My thought is that maybe it's okay to call the folder just `state` but to name the crate `trin-state` (assuming that's the name you see in crates.io). 
I'm thinking the same way, what seems important to me is the crate name in `Cargo.toml` (because of crates.io and collusion with other crate names). Folder naming is just personal preference, although most of the popular rust crates are naming folders same as the crates. 

> With `peertest`, and the idea being to test multiple clients eventually, maybe it should be named `portal-peertest` or `ethportal-peertest`, or something like that. Since it's a global name, it should stand well on its own, among _all_ rust crates.

Good point to have a global name for this crate, it is possible in the future even this to be in its own repo.